Dockerfile

```diff
@@ -115,11 +113,11 @@ RUN rm -vf /charts/*.sh /charts/*.md
 # This image includes any host level programs that we might need. All binaries
 # must be placed in bin/ of the file image and subdirectories of bin/ will be flattened during installation.
 # This means bin/foo/bar will become bin/bar when rke2 installs this to the host
-FROM rancher/k3s:v1.21.3-k3s1 AS k3s
-FROM rancher/hardened-kubernetes:v1.21.3-rke2r2-build20210809 AS kubernetes
-FROM rancher/hardened-containerd:v1.4.9-k3s1-build20210908 AS containerd
-FROM rancher/hardened-crictl:v1.19.0-build20210223 AS crictl
-FROM rancher/hardened-runc:v1.0.1-build20210908 AS runc
+FROM rancher/k3s:v1.22.1-rc1-k3s1 AS k3s
```

Comment:
```suggestion
FROM rancher/k3s:v1.22.1-k3s1 AS k3s
```

Reply:
We should bump all k3s/k8s versions to ~1.22.2 now yeah?

Reply:
yep, just haven't circled back since 1.22.2 came out. Will do that shortly.